src/gui/tray/ActivityItem.qml

```diff
@@ -67,6 +68,9 @@ MouseArea {
             Layout.fillWidth: true
             Layout.leftMargin: 40
             Layout.bottomMargin: model.links.length > 1 ? 5 : 0
+            // TODO this should be in the TalkReply Loader in ActivityItemContent
+            // but it did not work
```

Comment:
This is because the parent of the loader component is not a layout -- it is a `Column`, not a `ColumnLayout`

Reply:
I tried now, nothing bellow the text field moved. Will try again tomorrow.